server/src/databases/index.js

```diff
@@ -0,0 +1,49 @@
+const Sequelize = require("sequelize");
+require("dotenv").config();
+
+const config = require("../../config/index");
+
+const nodeEnv = process.env.NODE_ENV;
+const sequelizeProperties = config.connectionToDb[nodeEnv];
+
+const sequelizeConnection = () => {
+  const sequelizeUrlDB = new Sequelize(
+    sequelizeProperties.postgresDbName,
+    sequelizeProperties.postgresUser,
+    sequelizeProperties.postgresPassword,
+    {
+      host: sequelizeProperties.postgresHost,
+      dialect: sequelizeProperties.postgresDialect,
+    }
+  );
+  return sequelizeUrlDB;
+};
+
+const initialConnectionToDb = async () => {
+  let sequelizeUrlDB;
+
+  try {
+    sequelizeUrlDB = await sequelizeConnection();
+    await sequelizeUrlDB.authenticate();
+    console.log(
+      `Initial connection to ${sequelizeProperties.postgresDbName} has been established successfully.`
+    );
+    await sequelizeUrlDB.sync();
```

Comment:
?does syncing the models necessarily related to the connection to db? shouldn't it be in a separated function